### PR TITLE
Selenium: Change the 'GitStatusBar' to avoid unexpected 'stale element exception'

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Change the method _waitMessageInGitTab()_ in the _GitStatusBar_ to avoid the _'stale element reference'_ exception
* Correct the  methods in the _GitStatusBar_ using the _SeleniumWebDriverHelper_
* Delete the unused methods 

### What issues does this PR fix or reference?
#11553

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitStatusBar.java [hunk 1]
```diff
@@ -59,57 +60,41 @@ public GitStatusBar(SeleniumWebDriver seleniumWebDriver) {
...
   /** wait and click on remove button in the 'git console' */
   public void waitAndClickOnRemoveBtn() {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(ExpectedConditions.elementToBeClickable(gitStatusBarRemoveBtn));
-    gitStatusBarRemoveBtn.click();
+    seleniumWebDriverHelper.waitAndClick(gitStatusBarRemoveBtn);
   }
 
   /** click on the 'minimize 'button */
   public void clickOnStatusBarMinimizeBtn() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitStatusBar.java]: What about renaming `clickOnStatusBarMinimizeBtn` to `waitAndClickOnStatusBarMinimizeBtn`?

Comment (artaleks9) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitStatusBar.java]: It seems, this name is more exactly. OK, I'll change it.

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/965/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/965//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11590
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**